app/_src/gateway/how-kong-works/load-balancing.md

```diff
@@ -3,24 +3,30 @@ title: Load Balancing Reference
 ---
 
 Kong provides multiple ways of load balancing requests to multiple backend
-services: a straightforward DNS-based method, and a more dynamic ring-balancer
-that also allows for service registry without needing a DNS server.
+services: the default DNS-based method, and an advanced set of load-balancing
+algorithms using the Upstream entity.
 
-## DNS-based load balancing
+The DNS load balancer is enabled by default and is limited to round-robin
+load-balancing. The `upstream` entity has health-check and circuit-breaker
+functionalities, besides the more advanced algorithms like least-connections,
+consistent-hashing, and lowest-latency.
+
+Please read up on the [DNS caveats](#dns-caveats) depending on your infrastructure .
 
-When using DNS-based load balancing, the registration of the backend services
-is done outside of Kong, and Kong only receives updates from the DNS server.
+## DNS-based load balancing
 
 Every Service that has been defined with a `host` containing a hostname
 (instead of an IP address) will automatically use DNS-based load balancing
-if the name resolves to multiple IP addresses, provided the hostname does not
-resolve to an `upstream` name or a name in your DNS hosts file.
+if the name resolves to multiple IP addresses.
 
 The DNS record `ttl` setting (time to live) determines how often the information
 is refreshed. When using a `ttl` of 0, every request will be resolved using its
 own DNS query. Obviously this will have a performance penalty, but the latency of
 updates/changes will be very low.
 
+The (weighted) round-robin algorithm used depends on the DNS record type of the
```

Comment:
```suggestion
The weighted round-robin algorithm used depends on the DNS record type of the
```

Reply:
replaced this one slightly different, please have a look.